src/features/projectsV2/ProjectSection.tsx

```diff
@@ -0,0 +1,24 @@
+import ProjectSnippet from './components/ProjectSnippet';
+import style from './styles/ProjectSection.module.scss';
+import { useProjects } from './ProjectsContext';
+
+const ProjectSection = () => {
+  const { projects, isLoading, isError } = useProjects();
+  if (isLoading || isError) {
+    return null;
+  }
```

Comment:
Add in a loading state for the project list, with a skeleton loader.

Reply:
Done